resotolib/resotolib/baseplugin.py

```diff
@@ -211,6 +211,16 @@ def collect(self) -> None:
         """Collects all the Cloud Resources"""
         pass
 
+    @staticmethod
+    def update_tag(resource: BaseResource, key: str, value: str) -> bool:
+        """Update the tag of a resource"""
+        return resource.update_tag(key, value)
+
+    @staticmethod
+    def delete_tag(resource: BaseResource, key: str) -> bool:
```

Comment:
1) Is there a good reason to make those methods static?
To be more specific:  would it be possible to create plugin instances when the worker starts and maintain those instances for the lifetime of the worker?

2)  can we also add the config to the signature?